distribution/src/config/jvm.options

```diff
@@ -1,12 +1,29 @@
+################################################################
+##
 ## JVM configuration
+##
+################################################################
+##
+## WARNING: DO NOT EDIT THIS FILE. If you want to override the
+## JVM options in this file, or set any additional options, you
+## should create one or more files in the jvm.options.d
+## directory containing your adjustments.
+##
+## See https://www.elastic.co/guide/en/elasticsearch/reference/current/jvm-options.html
```

Comment:
I think we want to avoid a link here, since it could easily break in the future. It also will link users to the wrong version for their version. For example, linking to current would redirect a user to 8.0.0 when that's released, but maybe they're on 7.9 and we've made breaking changes.

Reply:
I'm not too worried about breaking this link, the docs team put quite some effort into avoiding broken links to `current` docs in future versions since it's so harmful to the searchability of our docs. Note that there's prior art: we already include a link to the current heap size docs further down this file.

I couldn't see a simple way to link to the correct version of these docs today, although this would be a nice feature to have.

Reply:
In the spirit of progress over perfection I merged this PR and opened https://github.com/elastic/elasticsearch/pull/61456#discussion_r475779439 regarding this comment.